packages/entando-apimanager/modules/state/api/reducer.js

```diff
@@ -3,6 +3,7 @@ import { SET_API } from './types';
 const initialState = {
   useMocks: true,
   domain: null,
+  defaultPath: '',
```

Comment:
this should really be named prefix, uriPrefix or urlPrefix, because that is what it is.
defaultPath makes people think that if no path is specified this is the one used

Reply:
every selector, action, etc referencing to it should get a new name as well